app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -97,6 +100,18 @@ protected void onCreate(final Bundle savedInstanceState) {
             LOGGER.info("Bluetooth handler not started, most likely Bluetooth is not enabled");
             this.enableBluetooth(false);
         }
+        if (intent.getSerializableExtra(ConstantKeywords.SUCCESSFUL_BLUETOOTH) != null) {
+
+            final boolean successfulBluetooth = (boolean) intent.getSerializableExtra(ConstantKeywords.SUCCESSFUL_BLUETOOTH);
+
+            if (successfulBluetooth) {
```

Comment:
Could you refactor this into a separate method? The `onCreate` method is getting a bit crowded 😄